css/yzecoriolis.css

```diff
@@ -1716,11 +1719,44 @@
 }
 .yzecoriolis.sheet.actor .sheet-body .critical-injuries-list .injury {
   align-items: center;
-  background: rgba(19, 27, 53, 0.45);
   border-radius: 3px;
   padding: 2px;
-  margin-bottom: 2px;
+  margin-bottom: 5px;
   height: 26px;
+  color: white;
+}
+.yzecoriolis.sheet.actor .sheet-body .critical-injuries-list .critical-injuries-bg {
```

Comment:
My understanding is that the css changes should be applied in the `.less` files in the `less` folder. This file is generated (by `npm build`) and will be overwritten.

Reply:
Hmm, I see that folder on Github but not in my installation. I don't have any experience with npm and .less files in this regard :/

Reply:
@AlexOkafor Can you answer the question if it is enough to modify the .css and not the .less and if not what I have to take into account in this regard?

Reply:
@IrishWolf you would have to edit the appropriate `.less` files. Anytime a `yarn gulp` is run the `.css` files are re-generated from the less files, which will blow away anything modified in css.   If you're running npm instead of yarn you can do a `npm run compile` to re-generate the files on demand or `npm run gulp` to have them watch the files and re-gen CSS as you edit `.less` files.

Reply:
> @IrishWolf you would have to edit the appropriate `.less` files. Anytime a `yarn gulp` is run the `.css` files are re-generated from the less files, which will blow away anything modified in css. If you're running npm instead of yarn you can do a `npm run compile` to re-generate the files on demand or `npm run gulp` to have them watch the files and re-gen CSS as you edit `.less` files.

Thx! I will look into it.
Do you have an idea why I don't have the .less-folder in my installation? (tryed Ubuntu 20.04 and win10)